### ISSUE TITLE
Design Intent - Accessibility Feedback - AuthExp, MyVA 3.0

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#109842]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

- Lots of good stuff here! I like the clean-up and tightening up happening ahead of broader revisions.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** I'm concerned about the right-hand column on desktop being missed by screen magnification users, eg. [Windows Magnifier](https://support.microsoft.com/en-us/windows/use-magnifier-to-make-things-on-the-screen-easier-to-see-414948ba-8b1c-d3bd-8615-0e5e32204198), [MacOS Zoom](https://support.apple.com/guide/mac-help/zoom-in-on-whats-onscreen-mchl779716b8/mac), and various third-party magnification tools (including physical magnifiers on the display). The behavior we would typically expect for a magnification user would be to mostly hang out in the left-hand column and move straight down the page, only shifting right when there's something to visually indicate that there's content over there that they should look at. Since some of the content in the cards is kind of sparse (lots of white space in the right-hand part of the card) and since the margin between left and right columns is fairly wide, a screen magnification user may not realize there's anything over there to look at.
   
   If it's possible, I'd encourage any user research studies to recruit for low vision users who use screen magnifiers. But that's a hard population to recruit for specifically, so if nothing else I'd encourage you to play with a magnifier yourself to see if there are changes you can make to the layout that would help. I think Ryan Thurlwell alluded to this, but there's no rule that My VA has to be two 50-50 columns, so there may be adjustments you can make to help the layout better meet the needs of the content.
- [ ] **Should:** I think it would be a good idea to document and account for every possible alert that can appear on My VA, from every possible source --- like, the loudest and most colorful iteration of My VA that a user could ever see. I'm especially thinking about heading structure, making sure that any alerts drop into the page in a way that fits into the heading hierarchy in a sensible way. But it's also worth taking this as an opportunity to think about visual weight, cognitive load, and focusing users on the most important thing on the page.
- [ ] **Consider:** I think there's some potential for confusion for "Helpful topics" and "Get help" as two adjacent sections. I don't think the confusion would be all that disruptive --- users should be able to figure out the purpose of each section relatively quickly --- but it's extra cognitive load that we shouldn't put on people if we don't have to.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
